presto-native-execution/presto_cpp/main/types/PrestoToVeloxQueryPlan.cpp

```diff
@@ -1731,7 +1731,7 @@ core::PlanNodePtr VeloxQueryPlanConverterBase::toVeloxQueryPlan(
   auto joinType = toJoinType(node->type);
 
   if (node->criteria.empty() && core::isInnerJoin(joinType) && !node->filter) {
-    return std::make_shared<core::CrossJoinNode>(
+    return std::make_shared<core::NestedLoopJoinNode>(
```

Comment:
What is this change about?

Reply:
It is a compatible change. We recently rename CrossJoinNode to NestedLoopJoinNode at Velox side.